automation/CommandSequence.py

```diff
@@ -0,0 +1,58 @@
+from Errors import CommandExecutionError
+
+class CommandSequence:
+    """A command sequence should be associated with one top url site visit"""
```

Comment:
This comment should describe the command sequence class. How it's useful and perhaps include sample code on how to use it. I think this is particularly important as this is a big change in the infrastructure's interface and the class is user-facing.